04_numeronym/mohankrishna/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	numeronyms := make([]string, len(vals))
+	for pos, value := range vals {
+		trimmedVal := []rune(strings.TrimSpace(value))
```

Comment:
Probably safer to use `strings.Trim(value, "\n\t ")`

Reply:
Updated it with `Trim(value, "\n\t ")`